lib/elsa/group/supervisor.ex

```diff
@@ -22,10 +30,13 @@ defmodule Elsa.Group.Supervisor do
     registry = registry(connection)
     group = Keyword.fetch!(init_arg, :group)
 
+    group_consumer_supervisor = {:via, Elsa.Registry, {registry, :group_consumer_supervisor}}
+
     children =
       [
         {DynamicSupervisor, [strategy: :one_for_one, name: {:via, Elsa.Registry, {registry, :worker_supervisor}}]},
-        consumer_supervisors(registry, topics, config),
+        {DynamicSupervisor, [strategy: :one_for_one, name: group_consumer_supervisor]},
```

Comment:
if the workers need an initializer, do they also need to be started under a dynamic_process_manager? Or do workers under a group-based consumer not need initializers?
Also, is there a need to have a separate dynamic supervisor for the workers under a group or can they share a dynamic supervisor with the group consumer?

Reply:
Going to look into this, made a separate for the moment just to avoid any conflict.